mappings/net/minecraft/util/annotation/FieldsAreNonnullByDefault2.mapping

```diff
@@ -0,0 +1,12 @@
+CLASS net/minecraft/class_6037 net/minecraft/util/annotation/FieldsAreNonnullByDefault2
```

Comment:
Is there any better naming for this class? Or do we not see the usages and therefore don't know where it's used

Reply:
You just look it up. There is no recognizable pattern on how this is used than `FieldsAreNonnullByDefault`, hence I go with this. If you have a name, please suggest it than commenting "please get a better name". Thanks!

Reply:
On a side note, the math method return nonnull one originally I named `2` suffix as well, but it is only used in that b3d math package, so I named it `Math...`. This annotation unfortunately has no such comparable trait and hence I chose such a bad name. @Juuxel might have some better ideas?

Reply:
In mojang code they might be `com.mojang.blaze3d.something.FieldsAreNonnullByDefault` etc if they're only used there, but I'm not sure whether we should do that in yarn.

Reply:
weird, it doesnt seem like it has any discernable use. i guess this is fine

Reply:
This one's usage is scattered with the other non null by default, like in item and tag packages at least.